### ISSUE TITLE
No Tif-Header Values if Process Title generated

### ISSUE THREAD

**Issue by m-kotzyba** [kitodo/kitodo-production#4466]
During the creation of new Process via OPAC, the Process Title (Vorgangstitel) generation does not set any values in the Tif-Header Image-Beschreibung.

Cf. Screenshot including the comparison to a properly working Kitodo 2 system.

Are there any adaptations in the (old) kitodo_projects.xml necessary or are adaptations in the kitodo_opac.xml, resp. pica2kitodo.xsl required? This is related to #3777.

![empty_tif_header](https://user-images.githubusercontent.com/85474095/121496513-d3e53700-c9da-11eb-901c-3af9d9c6467a.PNG)

**Comment by matthias-ronge** [comment]
Yes, if you use a ruleset that differs from the settings in the `kitodo_projects.xml` file, then you have to adapt the file. The file is a relic from Production version 2. A description (in German) is available [here](/kitodo/kitodo-production/wiki/Projekt-XML-Datei).

**Comment by m-kotzyba** [comment]
As far as I could figure out, not the item values (e.g. “Erscheinungsjahr”) but the metadata from the (new) ruleset (e.g. PublicationYear) have to be used in the tifheader section. At least it works for our system (cf. Screenshot 1)

![projects](https://user-images.githubusercontent.com/85474095/123259651-00c63d80-d4f5-11eb-97e2-03a77d796f8d.PNG)

However, it seems that the declaration in the `<item … > … </item>` section is ignored. 

Another issue is that the first line in the tifheader section 

`<monograph>'|[[DOC_TYPE]]'+docType+'|[[HAUPTTITEL]]'+TitleDocMain+'|[[AUTOREN/HERAUSGEBER]]'+ContributorPerson+'|[[JAHR]]'+PublicationYear+'|[[ERSCHEINUNGSORT]]'+PlaceOfPublication+'|[[VERZ_STRCT]]'+TSL_ATS+'_'+CatalogIDDigital+'|'</monograph>`

is taken always even if a multivolume is selected in the OPAC search. Is there a way to change this? (cf. Scheenshot 2 where a multivolume is selected but the monograph tifheader configuration is used).

![gui](https://user-images.githubusercontent.com/85474095/123259818-34a16300-d4f5-11eb-9068-37040d8db02c.PNG)

In contrast, the configuration according to different Project-Names is considered and used (otherwise the default configuration is used).

---
--> That is, it would be nice to know what values/sections/constrains from the `kitodo_projects.xml` are used and which are ignored in the end.
--> Finally, what is the right syntax to select sub-key values. For example, the ruleset has the id=Person and a sub-key with id=LastName. How to access this? “Person.LastName” or “Person:LastName” do not work.
